tap_google_analytics/client.py

```diff
@@ -125,9 +149,10 @@ def _make_request(self, method, url, params=None, data=None):
         else:
             response = self.session.request(method, url, headers=headers, params=params)
 
-        error_message = _is_json(response) and response.json().get("error", {}).get("message")
-        if response.status_code == 400 and error_message:
-            raise Exception("400 Client Error: Bad Request, details: {}".format(error_message))
+        response_is_json = _is_json(response)
+        error_message = response_is_json and response.json().get("error", {}).get("message")
+        if response.status_code in [400, 403] and error_message:
```

Comment:
Instead of in a list can you do in a set here? The difference in performance is negligible I'm sure but it looks cleaner to me...

Reply:
Sure that makes sense